DigixGlobal/governance-ui-components
# Feature/dgdg 381b

[FEATURE] Show wallet balances and USD values of tokens 

Re: [DGDG-383](https://tracker.digixdev.com/agiles/88-14/89-17?issue=DGDG-383)

This shows the Token Balances of (ETH, DGX and DGD once Address is Loaded) and once user visits the Wallet Page

**Test Plan**
- Load a wallet and verify that Tokens have value on USD section
- Go to Wallet Section and verify that tokens have USD equivalent values
- Load a wallet that has not locked a DGD or approved an interaction and verify if everything works as expected

**Note**
Design for the wallet Page is not done yet.
